DESCRIPTION

```diff
@@ -58,3 +58,5 @@ Encoding: UTF-8
 Language: en-US
 Roxygen: list(markdown = TRUE)
 RoxygenNote: 7.2.3
+Remotes: 
+    r-lib/rlang
```

Comment:
Since rlang 1.1.0 has been released, what is it here that requires installing the dev version?

Reply:
Whoops - `use_standalone("r-lib/rlang", "types-check")` did that, it slipped by me.